### DISCUSSION TITLE
type of language used for plain text in fenced code blocks(`text` is not supported by prism)

### DISCUSSION THREAD

**Discussion by k-rajat19** [PrismJS/prism#3570] [Q&A] [answered]
Hi. I want to ask what is the type of language I should use for plain text in fenced code blocks
I'm doing something like this in my markdown file
```
```text
Tom      8
Kris     2
Ahmad    5
Beau     6
dtype: int64
```
but got an error that `text` is recognized.
Also, I don't leave it without specifying any type because I've enabled [MD040 lint rule](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md040---fenced-code-blocks-should-have-a-language-specified).

**Comment by RunDevelopment** [comment, marked as answer]
What do you mean by you got an error? If a language isn't recognized, the code block simply remains unhighlighted.

**Reply by k-rajat19** [reply]
it was a lint error because of MD040 lint rule, I've fixed it by using `markdown` instead of `text` for now

**Reply by k-rajat19** [reply]
do you have any suggestions on this?

**Reply by RunDevelopment** [reply]
Prism is a syntax highlighter. We have nothing to do with any linting errors.
